src/server.js

```diff
@@ -100,7 +100,29 @@ server
         res.status(503).send()
       })
   })
-  
+
+  .get('/appointments/:confirm/:documentId', (req, res) => {
```

Comment:
why a confirm parameter?

Reply:
that's not what I meant 
   .get('/appointments/**:confirm**/:documentId', (req, res) => {
 ":confirm"  is a parameter that it is not used, so it should not be a parameter 
with the latest changes the code would not work